egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -57,8 +57,12 @@ def CheckIfCudaCompiled():
 
 
 class KaldiCommandException(Exception):
-    def __init__(self, command, err):
+    def __init__(self, command, err, stdout=None, code=None):
         Exception.__init__(self,"There was an error while running the command {0}\n".format(command)+"-"*10+"\n"+err)
+        self.stderr = err;
+        self.stdout = stdout;
+        self.command = command;
+        self.code = code
```

Comment:
I can help with implementing it, if Vijay is too busy with his thesis, but he should propose what would be the way to go.